apps/service_broadcast/lib/broadcast/init.ex

```diff
@@ -1,16 +1,22 @@
 defmodule Broadcast.Init do
+  use Annotated.Retry
+
   use Initializer,
     name: __MODULE__,
     supervisor: Broadcast.Stream.Supervisor
 
+  @retry with: constant_backoff(100) |> take(10)
   def on_start(state) do
-    Broadcast.Stream.Store.get_all!()
-    |> Enum.reject(&is_nil/1)
-    |> Enum.reject(&Broadcast.Stream.Store.done?(&1))
-    |> Enum.each(fn load ->
-      Broadcast.Stream.Supervisor.start_child(load)
-    end)
-
-    {:ok, state}
+    store = Broadcast.Stream.Store.get_all!()
+    case(store) do
+      {:error, _} -> {:error, "Broadcast failed to read state from store during startup"}
```

Comment:
the `{:error, reason}` tuple will never match here since `Broadcast.Stream.Store.get_all!()` is wrapping `Brook.get_all_values!()` which will only succeed and return the value or else raise an exception.

Reply:
Fixed for broadcast. I'll do the rest of the apps the same way if this looks good.